### ISSUE TITLE
It would be nice to include the Dark UI plugin

### ISSUE THREAD

**Issue by adrinux** [carloslfu/figma-app-linux#12]
Seems to be available for a custom build of the mac desktop version, so might be applicable to linux too.
https://www.papertiger.com/figma-dark-ui-plugin

**Comment by winstonford** [comment]
+1

**Comment by adalbertsen** [comment]
there is an unofficial plugin manager that contains the dark theme plugin too and other stuff that enhance your workflow: https://github.com/figma-plus/figma-plus can this be implemented in some way? they only have installers for the windows and mac desktop apps and a chrome plugin

**Comment by carloslfu** [comment]
Yeah, Figma Plus a great project and I think it is the way to go. The easiest way is to figure out what and where the https://github.com/figma-plus/installer is installing the Figma Plus code and create a build including it. We should have two builds one with Figma Plus and one without it.

**Comment by carloslfu** [comment]
For reference, the code that injects the plugins is here: https://github.com/figma-plus/installer/blob/master/renderer.js#L200.

**Comment by adalbertsen** [comment]
Sadly, they stopped developing Figma Plus, because the Figma team told them to - https://spectrum.chat/figma-plus/general/thank-you-and-good-bye~ef95c63c-b8e1-4885-8404-d8bf08fdb1d0

Although it won't be maintained, some kind of similar plugin installer/manager would be nice built into the app itself 😕

**Comment by carloslfu** [comment]
Oh, bad news. But there is something great about it, see https://spectrum.chat/figma/general/why-did-you-shut-down-figma-plus~b493857c-37cb-4d93-85a0-5abec5d82b84?m=MTU1ODQ1ODEzNDIwNQ==, It seems that Figma will prioritize the creation of an official Figma Plugins API.

**Comment by adalbertsen** [comment]
i hope that we can use plugins natively soon 😊

**Comment by winstonford** [comment]
I hope we have an opensource option soon, like https://github.com/akiraux/Akira

**Comment by adalbertsen** [comment]
i'm really hyped for Akira, but even when a minimum viable product will be out, it will have a lot of missing features 😕

**Comment by gusbemacbe** [comment]
Hi @carloslfu,

Figma team has released the new version, supporting plugins: https://www.figma.com/plugins-beta/

**Comment by carloslfu** [comment]
Good news @gusbemacbe!

**Comment by adalbertsen** [comment]
for me on ubuntu 19.04, this version works flawlessly with plugins and all 😁
